boltons/mathutils.py

```diff
@@ -0,0 +1,44 @@
+"""This module provides useful math functions on top of Python's built-in :mod:`math` module.
+"""
+
+
+def ceil_from_list(lst, x, allow_equal=True):
```

Comment:
Fair comment. I've refactored the function names with the suffix `_from_iter` in https://github.com/mahmoud/boltons/commit/3263f4414d08a1a9ea406891112da3fbeb903196. 

I thought about including this code in the `iterutils` module, but I didn't for the following reasons:
1. These functions are alternatives to `math.ceil()` and `math.floor()` in the standard library.
2. These functions return a single value, not an iterable.
3. There may be others with similar utils that do not suit the `boltons` sub-modules available to date.